source/views/guides/group-by-and-queryable-by-name.html.slim

```diff
@@ -0,0 +1,118 @@
+section.banner
+  .content-wrapper
+    .banner-wrapper
+      h2.banner__heading Group By and QueryableByName
+
+main
+  section.demo
+    .content-wrapper
+      .guide-wrapper
+        markdown:
+          Diesel can't statically check all queries due to the limitless
+          combinations of SQL. When you can't represent your query with
+          Diesel's DSL, you can fallback to
+          [`sql_query`](http://docs.diesel.rs/diesel/fn.sql_query.html).
+          Aggregations such as `GROUP BY` provide a great learning example for
+          turning raw SQL into a structure Diesel can work with.
+
+          **This guide assumes that you're using PostgreSQL.** Before we start,
```

Comment:
s/Before we start,/If you want to follow along

Reply:
Ok the getting started page contains this language -- I'll take this as my opportunity to update the sentence with the note that while the guide assumes postgresql for the data types, `sql_query` can be used across all backends.

Reply:
Is there anything in this guide that is PG specific? Can we just remove this bit entirely?

Reply:
Timestamp is only implemented for PG and Sqlite. If you want I can buck using the `posts` table and come up with a table that can be used on all backends. Also when implementing `QueryableByName`, if one wants to eschew adding the type signature adding the specific backend makes it little more concise. But I can go either way.

This reminds me, I need to clean up my reference to Timestamptz and `QueryableByName<Sqlite>` 🙄

Reply:
`Timestamp` is implemented for all backends. The docs stating otherwise were out of date.